torchrl/lightning/ppo.py

```diff
@@ -0,0 +1,123 @@
+"""Template for a PPO model on the pendulum env, for Lightning."""
+
+__all__ = ["PPOPendulum"]
+
+import typing as ty
+
+import torch
+from tensordict import TensorDict  # type: ignore
```

Comment:
why the 
```
# type: ignore
```
?

Reply:
My `mypy` in my VSCode was complaining... So I mypy-ignored the line. This is not necessary in the codebase.